section2-week1/exercise 8/Main.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+* Exercise 8 - Magic 8 Ball
+Write a program that will predict the future for the user. Tell them to ask a question and then create a prediction for them. Make sure to use an enum and a switch statement for this exercise
+*/
+#include <iostream>
+#include <stdio.h>      /* printf, scanf, puts, NULL */
+#include <stdlib.h>     /* srand, rand */
+#include <time.h>  
+
+using namespace std;
+
+//value definition
+
+
+int main()
+{
+	//enum variables
+	enum answers {
+		yes, no, maybe, perhaps
+	};
+	//inicialize variables
+	answers ans;
+	ans = yes;
+
+
+	int v1;
+
+	//selecting the answer
+	srand(time(NULL));
+	v1 = rand() % (1 - 5);
+
+	char ask[100];
+	cout << "Would you like to know what happens in your future?" << endl;
+	cout << "\n+++++++++++++++++++++++++++++++++++++++++++++++++++" << endl;
+	cout << "\nYou can ask to me any question... " << endl;
+	cin >> ask;
+	
+
+	
+
+	//giving and answer to the number random
+	switch (v1)
```

Comment:
There is no need for two variables and two switches here. You can compare v1 to the enum values directly.